deps/molecule/VERSION

```diff
@@ -1 +1,2 @@
-0.4.1
+base on molecule 0.4.1
```

Comment:
This version file is so strange, can we just use a new version number?

Reply:
1. Current C headers are really different (for example, the `Decorator`was added after we locked them in this repository), so I cherry pick the patch base on our locked version.

   Could I just remove this `VERSION` file, since we have locked them in this repository.
   Or I changed this file to `molecule 0.4.1 + commit f714c99905ff148f6088f905c6579e78aa7d0995`

2. Rust SDK and C APIs for Molecule use a same git repository.

   Should I publish a new version of rust crate when I just increase the version number for C part only?
   Or I have to split them into two repositories?

Reply:
Ouch somehow I missed it here, so the `VERSION` file is not that important, what's important, is that I want us to document a way, so anyone can generate the exact same version of the molecule files used here. In other words, we should ensure as much of our code in the repository follows reproducible build principles.

Reply:
`molecule 0.4.1 + commit f714c99905ff148f6088f905c6579e78aa7d0995` +1